sass/base/_helpers.scss

```diff
@@ -16,10 +16,22 @@
   color: $grey;
 }
 
+.font-sans-serif {
```

Comment:
Abbiamo appena introdotto le [stesse classi su Bootstrap Italia](https://italia.github.io/bootstrap-italia/docs/contenuti/tipografia/#famiglie-di-caratteri), nominandole però `text-` per coerenza con gli altri modificatori. Magari può tornare utile in futuro anche qui usare gli stessi nomi.